### PR TITLE
chore: add trusted-contribution.yml

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by apeabody** [comment]
@Shabirmean - Automate this as well

File: .github/trusted-contribution.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+annotations:
+  - type: comment
+    text: "/gcbrun"
+trustedContributors:
+  - release-please[bot]
+  - renovate[bot]
+  - forking-renovate[bot]
+  - dependabot[bot]
+  - apeabody
```
Comment (Shabirmean) [inline .github/trusted-contribution.yml:23]: @apeabody - Can we try replacing username with `@GoogleCloudPlatform/point-of-sale`?

Comment (Shabirmean) [reply .github/trusted-contribution.yml:23]: Also can you accept the invite for the 2 groups I've sent?

**Review by Shabirmean — APPROVED** [review]

**Comment by Shabirmean** [comment]
@apeabody - can you make an empty commit push to test this?

**Review by Shabirmean — APPROVED** [review]

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**35.202.68.113**](http://35.202.68.113)
- **Embedded H2** DB backed deployment: [****](http://)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**35.202.68.113**](http://35.202.68.113)
- **Embedded H2** DB backed deployment: [**34.29.134.138**](http://34.29.134.138)

**Pull request merged**